pandas/core/window/rolling.py

```diff
@@ -478,13 +495,31 @@ def _get_window_indexer(self, window: int) -> BaseIndexer:
             return VariableWindowIndexer(index_array=self._on.asi8, window_size=window)
         return FixedWindowIndexer(window_size=window)
 
+    def _apply_series(self, homogeneous_func: Callable[..., ArrayLike]) -> "Series":
+        """
+        Series version of _apply_blockwise
+        """
+        _, obj = self._create_blocks(self._selected_obj)
+        values = obj.values
```

Comment:
Question: Did we want to move away from `.values`? Should we use `to_numpy()` instead?

Reply:
We do, but doing that separately.  This PR is intended to keep behavior unchanged